_specs/ecip-1055.md

```diff
@@ -1,17 +1,15 @@
 ---
 ecip: 1055
 title: Flyclient - Succinct PoW Using Merkle Mountain Ranges
-author: Zac Mitton @zmitton
+author: Zac Mitton (@zmitton)
 discussions-to: https://github.com/ethereumclassic/ECIPs/issues/144
 status: Draft
 type: Standards Track
-category (*only required for Standard Track): Core
+category: Core
 created: 2019-03-12
-replaces (*optional): 210, 1094, 641
-
 ---
 
-## Simple Summary
+## Simple Summaryhttps://github.com/etclabscore/ECIPs/blob/master/ECLIPs/ECIP-draft_succinct-proofs.md
```

Comment:
## [Simple Summary](https://github.com/etclabscore/ECIPs/blob/master/ECLIPs/ECIP-draft_succinct-proofs.md)

Should this be a url?

Reply:
i don understand the question. i don't see that in the PR

do you mean to add that URL as a reference? if so, I don't think so because the ECIP is copied in its entirety

Reply:
unless @zmitton want to reference it?

Reply:
I was referring to this commit https://github.com/ethereumclassic/ECIPs/pull/145/commits/3b2f6b5c78f7c219d20f2febaa300c05fa7aa90b 

but it seems to have been reverted in the following commit.  ignore me